index.html

```diff
@@ -5,13 +5,49 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles/index.css">
-    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="index.css">
+    <title>gabriel's portfolio</title>
 </head>
 
 <body>
-    <h1 class="first">Hello microverse</h1>
+    <div class="container">
+        <header>
+            <div class="single-shape">
+                <div class="shapes triangle">
 
+                </div>
+                <div class="shapes square">
+
+                </div>
+            </div>
+            <nav>

```

Comment:
- [ ] Add the logo on the left side and our logo should have `a` tag.
![port6](https://user-images.githubusercontent.com/55635977/129165689-6b3ad831-bf3f-4cec-8126-b50ff2cfed1b.png)